### ISSUE TITLE
False-positive "Match must be exhaustive" in patterns with `..`

### ISSUE THREAD

**Issue by vlad20012** [intellij-rust/intellij-rust#3958]
```rust
struct S {
    s: String,
    e: E
}

enum E { A, B }

fn foo(s: S) {
    match s {
        S { e: E::A, s } => {}
        S { e: E::B, .. } => {}
    }
}
```

![image](https://user-images.githubusercontent.com/3221931/58910057-4b619b00-871d-11e9-8264-df0361c688ee.png)

**Comment by joelpalmer** [comment]
I also had this issue. But, the latest [update](https://intellij-rust.github.io/2019/06/20/changelog-100.html) seems to have resolved it for me. I have not looked through the commits but I can no longer repro the issue after the update. 

- CLion 2019.1.4
- Build #CL-191.7479.33, built on May 30, 2019
- macOS 10.14.5
